docs/standard/library-guidelines/breaking-changes.md

```diff
@@ -46,15 +46,17 @@ Because a source breaking change is only harmful when the developer recompiles t
 
 ### Behavior breaking change
 
-Behavior changes are the most common breaking change: almost any change in behavior could break someone. Even a bug fix can qualify if users relied on the previously broken behavior.
+Behavior changes are the most common breaking change: almost any change in behavior could break someone. Changing the results a method produces, the exception your library throws, or the data output it writes could all negatively impact applications and libraries than depend on you. Even a bug fix can qualify if users relied on the previously broken behavior.
 
-**✔️ CONSIDER** leaving new features off by default if they affect existing users, and let developers opt in to the feature with a setting.
+An approach to behavior breaking changes is to hide them behind settings. Settings let developers update to the latest version of your library while at the same time choosing to opt-in or opt-out of breaking changes. For example, ASP.NET Core MVC has the concept of a [compatibility version](https://docs.microsoft.com/en-us/aspnet/core/mvc/compatibility-version) that modifies the features enabled and disabled on `MvcOptions`.
```

Comment:
@rynowak Is that accurate? Anything you think worth adding?

Reply:
Yeah I think this is a really effective section.

The one addition that might help motivate this would be a fest sentences like this:

"Adding features and improving bad behaviors is a good thing, but without care it can make it very hard for existing software to upgrade. A strategy that lots consuming code adapt over team allows a library author to add value which bringing existing users along."

We did this partly out of empathy for existing apps, and partly out of a desired to make changes. Without a strategy to manage it - we would find ourselves **unable** to make improvements.

Reply:
Love it